content/applications/hr/payroll/configuration.rst

```diff
@@ -0,0 +1,315 @@
+Structure Types`.
+
+There are two default structure types configured in Odoo: *Employee* and *Worker*. Typically,
+*Employee* is used for salaried employees, which is why the wage type is *Monthly Fixed Wage*,
+and *Worker* is typically used for employees paid by the hour, so the  wage type is *Hourly*.
+
+.. image:: configuration/structure-type.png
+   :align: center
+   :alt: List of all structure types.
+
+Click the :guilabel:`Create` button to mae a new structure types. Most fields are pre-populated, but
+all fields can be edited. Once the fields are edited, click the :guilabel:`Save` button to save the
+changes, or click :guilabel:`Discard` to delete the entry.
+
+.. image:: configuration/new-structure.png
+   :align: center
+   :alt: New structure type box.
+
+Structures
+----------
+
+:guilabel:`Salary Structures` are the individual ways an employee gets paid, and are defined by
+various rules. Each :ref:`payroll/structure-types` lists all the *structures* available for that particular
+structure type.
+
+The amount of structures a company needs depends on how many different kinds of employees are
+working, and how different their pay is calculated. For example, a common structure that could be
+useful to add may be a **contract** worker, or an **intern**.
+
+To view all the various structures for each structure type, go to :menuselection:`Payroll -->
+Configuration --> Structures`.
+
+.. image:: configuration/salary-structure.png
+   :align: center
+   :alt: All available salary structures.
+
+Each structure lists the various *Structure Types* associated with that specific structure. Each
+record listed for a structure is a specific rule for that particular structure.
+
+The default structures for the Employee are *Regular Pay*, and *13th month - End of the year
+bonus*. Each specific structure lists how many rules that structure has. For example, *Regular Pay*
+has 11 rules. These rules are what calculate the payslip for the employee.
+
+.. image:: configuration/structure-regular-pay-rules.png
+   :align: center
+   :alt: Salary structure details for Regular Pay.
+
+Rules
+-----
+
+Each structure type has a set of rules to follow for accounting purposes. These rules are configured
+by the localization, and affect the *Accounting* application, so modifications to the default rules,
+or the creation of new rules, should only be done when necessary.
+
+To view all the rules, go to :menuselection:`Payroll --> Configuration --> Rules`. Click on the
+drop-down arrow next to a listed structure (such as :guilabel:`Regular Pay`) to view all the rules
+for that specific structure.
+
+.. image:: configuration/rules.png
+   :align: center
+   :alt: Rules for each salary structure type.
+
+Rule parameters
+---------------
+
+Salary *rule parameters* affect the various rules for salary configuration. New rule parameters are
+not recommended to create unless specifically needed by the accounting department. To access the
```

Comment:
After discussing with Zac, I changed this to a section explaining that it is not fleshed out and is mostly for Belgium. I did the same thing with the Resume section since it's similar- doesn't make a lot of sense, doesn't benefit the end user.